validate_depctrl.py

```
@@ -59,15 +63,22 @@ def check_changelog(lines, changelog, filename):
 
 def check_contents(entry, basename):
     all_fine = True
-    for ch in entry['channels'].values():
-        for f in ch['files']:
-            filename = basename + f['name']
-            with open(filename, 'rb') as fo:
+    for ch in entry["channels"].values():
+        for f in ch["files"]:
+            filename = basename + f["name"]
+            with open(filename, "rb") as fo:
                 content = fo.read()
-                content_lines = content.decode("utf-8").replace("\r", "").split("\n")
-            hash_good = check_hash(content, f['sha1'], filename)
-            version_good = check_version(content_lines, ch['version'], filename)
-            changelog_good = check_changelog(content_lines, entry['changelog'], filename)
+                try:
+                    content_lines = (
+                        content.decode("utf-8").replace("\r", "").split("\n")
+                    )
+                except:
```

do not use bare except